tomni/contour_operations/roundness/main.py

```diff
@@ -0,0 +1,24 @@
+from typing import Tuple
+
+import cv2
+import numpy as np
+
+
+def roundness(contour: np.ndarray) -> float:
+    """
+    Return roundess of the contour
+    Roundness is terminend by difference in areas of the contour and enclosing circle
```

Comment:
add also complete formula in here

Reply:
How would you do that without being longer then the 4 lines of code?

Reply:
roundness = area /r**2*pi